masterthesis_larsgabriel/README_CONTENTS.doc

```diff
@@ -0,0 +1,23 @@
+This README file is part of the master thesis:
```

Comment:
I removed the README_CONTENTS.doc because it was only meant for the master thesis. 
I moved README_ppsimscore.txt to the docs/ directory. Is this the correct location for the readme?
I removed example 2 and 3, i moved example 1 to the folder examples/sim-score/. Should there even be an example for pp_simscore or should I include more than 1 example?

Reply:
Yes: docs/README_ppsimscore.txt 
Yes, please put one example in that directory. We don't have a paper about it and it is not advertised. Therefore, currently it is practically hidden, even though it will be distributed with the code. We can change that with the first publication.